### PR TITLE
Update and rename Rockstar-Games.xml to rockstargames.com.xml

### PR DESCRIPTION
**Pull request by Folant**
Fix for #8229

### CODE REVIEW

**Comment by gloomy-ghost** [comment]
Thanks. Please:

Add targets:
- [ ] activate.rockstargames.com
- [ ] lyncdiscover.rockstargames.com
- [ ] m.rockstargames.com
- [ ] mail.rockstargames.com
- [ ] mindlink.rockstargames.com
- [ ] remote.rockstargames.com
- [ ] rsgedi-vpn.rockstargames.com
- [ ] socialclub.rockstargames.com
- [ ] de.socialclub.rockstargames.com
- [ ] es.socialclub.rockstargames.com
- [ ] es-mx.socialclub.rockstargames.com
- [ ] fr.socialclub.rockstargames.com
- [ ] it.socialclub.rockstargames.com
- [ ] ja.socialclub.rockstargames.com
- [ ] ko.socialclub.rockstargames.com
- [ ] pl.socialclub.rockstargames.com
- [ ] pt.socialclub.rockstargames.com
- [ ] ru.socialclub.rockstargames.com
- [ ] zh.socialclub.rockstargames.com

**Comment by gloomy-ghost** [comment]
Retest

File: src/chrome/content/rules/rockstargames.com.xml [hunk 1]
```diff
@@ -25,7 +25,6 @@
 	<target host="support.rockstargames.com" />
 	<target host="supportfiles.rockstargames.com" />
 	
-	<rule from="^http://(de|es|es-mx|fr|it|ja|ko|pl|pt|ru|zh)\.socialclub\.rockstargames\.com/" 
-		  to="https://socialclub.rockstargames.com/" />
+	<rule from="^http://(de\.|es\.|es-mx\.|fr\.|it\.|ja\.|ko\.|pl\.|pt\.|ru\.|zh\.)?socialclub\.rockstargames\.com/" to="https://socialclub.rockstargames.com/" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/rockstargames.com.xml]: This will cause looping redirection, can you revert it to 09b231f?

**Comment by gloomy-ghost** [comment]
Ping @Hainish for the failing Travis CI of 09b231f

**Comment by Hainish** [comment]
Strange, let's see if this is a temporary failure.  Restarting fetch test.

**Comment by gloomy-ghost** [comment]
Retest

**Comment by gloomy-ghost** [comment]
@Hainish Still

**Comment by Hainish** [comment]
Restarting job again

**Comment by Hainish** [comment]
Interesting.  For the failing `XX.socialclub.rockstargames.com` URLs, we're getting a series of redirects that eventually lands on http://socialclub.rockstargames.com.  This URL *does not respond* unless the header `Connection: keep-alive` is sent in this as well as previous requests.  So in modern browsers, the redirect chain works, but not in `curl`.

**Comment by Hainish** [comment]
I'm not sure why `remote.rockstargames.com` is failing.  I'm getting a

```shell
user@https-everywhere ~/workspace/https-everywhere (Folant-rockstargames.com) $ curl -LI https://remote.rockstargames.com/
curl: (60) SSL certificate problem: unable to get local issuer certificate
```

Even when I specify `--cacert` with the newest `cacert.pem` from https://curl.haxx.se/ca/cacert.pem, same result:

```shell
user@https-everywhere ~/workspace/https-everywhere (Folant-rockstargames.com) $ curl --cacert $(pwd)/cacert.pem -LI https://remote.rockstargames.com/
curl: (60) SSL certificate problem: unable to get local issuer certificate
```

Regardless, the certificate is valid in all supported browsers.

**Pull request closed without merging**